src/main/java/org/infernalstudios/shieldexp/init/ShieldDataLoader.java

```diff
@@ -47,7 +49,7 @@ public void apply(Map<ResourceLocation, JsonElement> files, ResourceManager reso
         FILE_MAP = files;
 
         for (ResourceLocation name : FILE_MAP.keySet()) {
-            if (ForgeRegistries.ITEMS.containsKey(name) || name.toString().equals(ShieldExpansion.MOD_ID + ":default")) {
+            if (ForgeRegistries.ITEMS.containsKey(name) || name.toString().equals(DEFAULT_SHIELD_NAME)) {
```

Comment:
See suggestion above (lines 32-35)
```suggestion
            if (ForgeRegistries.ITEMS.containsKey(name) || name.equals(DEFAULT_SHIELD_NAME)) {
```